src/core/zowe/core_for_zowe_sdk/config_file.py

```diff
@@ -238,7 +238,11 @@ def load_profile_properties(self, profile_name: str) -> dict:
         from the profile dict and populate fields from the secure credentials storage
         """
         try:
-            props = self.profiles[profile_name]["properties"]
+            props = self.profiles
+            lst = profile_name.split(".")
+            for i in range(len(lst)-1):
+                props = props[lst[i]]["profiles"]
+            props = props[lst[len(lst)-1]]["properties"]
```

Comment:
I still need to do some testing manually, but will this implementation retain the  `A.properties` or always go for the `LAST.properties` (in a structure like `A.B.C...LAST`)?

Reply:
It will give something like this
![2023-01-06_215517-](https://user-images.githubusercontent.com/75474786/213755909-3d35213e-58e7-440f-afde-8590cfc20c8f.png)